Dockerfile-linux.template

```
@@ -45,9 +45,11 @@ RUN set -eux; \
 	\
 # download/install js-yaml
 	mkdir -p /opt/js-yaml/; \
-	wget -O /opt/js-yaml/js-yaml.js "https://github.com/nodeca/js-yaml/raw/${JSYAML_VERSION}/dist/js-yaml.js"; \
-	wget -O /opt/js-yaml/package.json "https://github.com/nodeca/js-yaml/raw/${JSYAML_VERSION}/package.json"; \
-	ln -s /opt/js-yaml/js-yaml.js /js-yaml.js; \
+	wget -O /opt/js-yaml/js-yaml.tgz https://registry.npmjs.org/js-yaml/-/js-yaml-${JSYAML_VERSION}.tgz; \
+	echo "662e32319bdd378e91f67578e56a34954b0a2e33aca11d70ab9f4826af24b941 */opt/js-yaml/js-yaml.tgz" | sha256sum -c -; \
```

Unnecessary use of double quotes: :eyes:

```suggestion
	echo '662e32319bdd378e91f67578e56a34954b0a2e33aca11d70ab9f4826af24b941 */opt/js-yaml/js-yaml.tgz' | sha256sum -c -; \
```

Alternatively, we could keep the double quotes and move the checksum to live up with the version number that actually influences it (`ENV JSYAML_SHA256 ...`), which is probably a better change.

---

> we [should] keep the double quotes and move the checksum to live up with the version number that actually influences it (`ENV JSYAML_SHA256 ...`)

---

Done!